macrosynergy/pnl/historic_portfolio_volatility.py

```diff
@@ -457,5 +487,22 @@ def historic_portfolio_vol(
         start=start,
         end=end,
     )
-    print(df_vol.head(10))
-    print(df_vol.tail(10))
\ No newline at end of file
+
+    df_copy_vol: pd.DataFrame = historic_portfolio_vol(
+        df=df_copy,
+        sname="STRAT",
+        fids=fids,
+        est_freq="m",
+        lback_periods=15,
+        lback_meth="ma",
+        half_life=11,
+        rstring="XR",
```

Comment:
`rstring` and `sname` should be renamed for clarity

Reply:
So `return_string`, and  `signal_name` essentially. Makes sense to me.